copier.yml

```diff
@@ -78,7 +78,12 @@ ci:
 
 odoo_test_flavor:
   type: str
-  default: Both
+  default: |-
+      {% if odoo_version < 13.0 -%}
+          OCB
+      {%- else -%}
+          Both
+      {%- endif %}
```

Comment:
Thanks for the suggestion but I have managed to try it locally and it works already, I only have to deal with `pre-commit` because I hadn't it installed in the repository